models/simple_gain/model.json

```diff
@@ -0,0 +1,51 @@
+{
+    "system": {
+        "target": "audiomini",
+        "sampleClockFrequency": 48000,
+        "systemClockFrequency": 98304000,
+        "processing": "channel",
+        "audioIn": {
+            "wordLength": 32,
+            "fractionLength": 28,
+            "signed": true,
+            "numberOfChannels": 2
+        },
+        "audioOut": {
+            "wordLength": 32,
+            "fractionLength": 28,
+            "signed": true,
+            "numberOfChannels": 2
+        }
+    },
+
+    "devices": [
+        {
+            "name": "simple_gain",
+            "registers": [
+                {
+                    "dataType": {
+                        "fractionLength": 24,
+                        "signed": false,
+                        "type": "fix32",
+                        "wordLength": 32
```

Comment:
We could consider making this data type smaller, as we don't really need all of these bits for a gain; I doubt we would hear any differences at 2^-24...

Reply:
Thats fair.